lib/modules/userInfo.js

```diff
@@ -101,61 +73,131 @@ module.options = {
 export const highlightedUsers = {};
 
 module.go = () => {
-	if (module.options.hoverInfo.value) {
-		$(document.body).on('mouseover', UserTagger.usernameSelector, handleMouseOver);
-	}
+	watchForFutureChildren(document.body, '.author-tooltip', card => {
+		renderHoverCardInfo(card);
+	});
 };
 
-function handleMouseOver(e: Event) {
-	const authorLink = downcast(e.target, HTMLAnchorElement);
-	if (!UserTagger.usernameRE.test(authorLink.href)) return;
-	const [, username] = UserTagger.usernameRE.exec(authorLink.href);
-	if (!username) console.error(i18n('userInfoInvalidUsernameLink'));
-	const thing = Thing.from(authorLink);
+async function renderHoverCardInfo(hoverCard) {
+	const authorLink = await waitForDescendant(hoverCard, '.author-tooltip__profile-link');
+	const [, username] = UserTagger.usernameRE.exec(authorLink.getAttribute('href'));
+	applyResData(hoverCard, username);
+}
 
-	Hover.infocard(module.moduleID)
-		.target(e.target)
-		.options({
-			width: 475,
-			openDelay: module.options.hoverDelay.value,
-			fadeDelay: module.options.fadeDelay.value,
-			fadeSpeed: module.options.fadeSpeed.value,
-		})
-		.populateWith(card => showUserInfo(card, username, thing))
-		.begin();
+/**
+ * applyResData
+ * Apply additional RES data to the reddit hover-cards
+ *
+ * @param Element hoverCard
+ * @param string username
+ */
+async function applyResData(hoverCard, username) {
+	// Get main elements
+	const header = hoverCard.querySelector('.author-tooltip__head');
+	const links = hoverCard.querySelector('.author-tooltip__link-list');
+
+	// Render tagger
+	const userTag = await attachUserTagger(header, username);
```

Comment:
To avoid adding too much visual noise, the tagger probably ought to be placed next to the username.  `userTag.link` doesn't seem to fit well inside the new userInfo, so the link can probably be ignored as it seemingly is of low value.

The `attachUserTagger` function can then pretty much be replaced with:
```
UserTagger.applyToUser(
	downcast(header.querySelector('.author-tooltip__profile-link'), HTMLAnchorElement),
	{ renderTaggingIcon: true, renderVoteWeight: false }
);
```

Reply:
Possibly - next to the name felt more natural since its with reddits own additional icons get added (gold, cake day etc), vs next the the /u/etc on the full profiles :/

Reply:
A issue with placing it on the next line is that the native userTagger is position as if it is fixed height
![image](https://user-images.githubusercontent.com/1748521/32983032-ae717760-cc8e-11e7-81ee-f99002541240.png)

Reply:
I'll ask about implementing a frontend API method for "fix repositioning of reddit user card".